src/frontends/onnx/frontend/include/onnx_import/core/node.hpp

```diff
@@ -192,3 +192,11 @@ inline std::ostream& operator<<(std::ostream& outs, const Node& node) {
 }  // namespace onnx_import
 
 }  // namespace ngraph
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+using ngraph::onnx_import::Node;
```

Comment:
This is something we shouldn't do. You're introducing a symbol that will be deprecated to the `ov` namespace. Also, the `onnx_import::Node` is an implementation detail that should not be used outside of the onnx_importer. We can discuss it offline if needed.

Reply:
How we can avoid using `ngraph::onnx_import::Node` in public?
Currently, ctor of `frontend::onnx::NodeContext` expect this object, if we just use forward declaration of `ngraph::onnx_import::Node`, is this also an issue?

Reply:
discussed offline. 
decided to delete this `using `and instead use forward declaration of `ngraph::onnx_import::Node `

Reply:
done